psi4/src/psi4/lib3index/dfhelper.h

```diff
@@ -304,6 +304,9 @@ class PSI_API DFHelper {
     /// clear transformations
     void clear_transformations(); 
 
+    /// clear in-core AO integral
+    void clear_AO();
```

Comment:
Why define this function if you never call it?

Reply:
This was originally in `psi4/src/psi4/libsapt_solver/fdds_disp.cc`, it was created so that the program can clear the AO buffer outside of the dfhelper module. It was once needed during one of the attempts to fix the MO disk I/O issue, but it's not needed anymore in the final version (clearing AO buffer is done within `transform()` of the dfhelper module instead). Should I still keep this function here?

Reply:
No. If you don't use a function and don't plan to use the function, don't keep it.